### PR TITLE
Tools: clang format of templates, #727

### PR DESCRIPTION
**Pull request by martinfalk**

### CODE REVIEW

File: tools/meta/templates/moduledefine.h [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-	#else
-		#ifdef _WIN32
-			#define {{ module/api }} __declspec(dllimport)
-		#else
-			#define {{ module/api }}
-		#endif
-	#endif
+#ifdef INVIWO_ALL_DYN_LINK  // DYNAMIC
+// If we are building DLL files we must declare dllexport/dllimport
+#ifdef IVW_MODULE_{{ upper(module/name) }}_EXPORTS
```
Comment (ResearchDaniel) [inline tools/meta/templates/moduledefine.h:5]: Hmm, probably should ignore clang format here? The previous formatting made more sense in my opinion

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]

**Review by r-englund — APPROVED** [review]

File: tools/meta/templates/moduledefine.h (hunk 1 above)
Comment (petersteneteg) [reply tools/meta/templates/moduledefine.h:5]: I agree with Daniel, since this file is all macro definitions, and since it is quite hard to follow the logic without the indentation, I would prefer to keep it like that.. guess we should add the format guards though... Seems CMake also can auto generate these now... so maybe we should use that instead at some point...

Comment (martinfalk) [reply tools/meta/templates/moduledefine.h:5]: Then please go ahead and fix this.

Comment (martinfalk) [reply tools/meta/templates/moduledefine.h:5]: so far, no one cared what happened after the initial commit of such files and the subsequent clang format. It's not like these files will be ever looked at again.
